site/book/06-deploying-packages/01-initializing-a-package-for-apply.md

```diff
@@ -0,0 +1,36 @@
+Before you can apply the package to the cluster, it needs to be initialized
+using `live init`. This is a one-time client-side operation that adds metadata
+to the `Kptfile` specifying a `ResourceGroup` resource on the cluster associated
```

Comment:
This sentence is a bit vague. Can we be a bit more concrete about the information in the Kptfile rather than just say metadata. And be a bit more specific about the `ResourceGroup` resource than just saying it is "associated" with the package.

Reply:
Seeing now that this is explained in some more detail below.